tests/testthat/test-monitor.R

```diff
@@ -0,0 +1,30 @@
+test_that("getMonitorLevel works:", {
+  expect_equal(getMonitorLevel("verbose_tree"), MONITOR_LEVEL_VERBOSE_TREE)
+  expect_equal(getMonitorLevel("basic_tree"), MONITOR_LEVEL_BASIC_TREE)
+  expect_equal(getMonitorLevel("basic"), MONITOR_LEVEL_BASIC)
+  expect_equal(getMonitorLevel("none"), MONITOR_LEVEL_NONE)
+})
+
+test_that("log_metaepoch_snapshot works:", {
+  metaepoch_snapshot <- methods::new(
+    "MetaepochSnapshot",
+    demes = list(),
+    best_fitness = 0,
+    best_solution = 0,
+    time_in_seconds = 0,
+    fitness_evaluations = 0,
+    blocked_sprouts = list(),
+    is_evolutionary = TRUE
+  )
+  root_id <- 0
+  metaepochs_count <- 1
+  monitor_level <- MONITOR_LEVEL_BASIC
+  expect_output(
```

Comment:
To sprawdza po prostu czy cokolwiek się wyprintowało? xdd ..w sumie najważniejsze, że test coverage skacze

Reply:
tak, nie miałem pomysłu jak mogę to testować

Reply:
jak znajdziesz jakieś lepsze rozwiązanie to możesz poprawić